.jenkins/build.sh

```diff
@@ -36,12 +36,17 @@ main() (
     declare -a LIST_FILE_ARRAY=( $(find $(pwd) -type f -name 'jacoco.xml') )
     LIST_FILE=$(IFS=, ; echo "${LIST_FILE_ARRAY[*]}")
     # Why sonar plugin is not declared in pom.xml: https://blog.sonarsource.com/we-had-a-dream-mvn-sonarsonar
+    # TODO Disabling Sonar cache due to "ClassNotFoundException: com.google.common.collect.LinkedListMultimap" issue
```

Comment:
```suggestion
    # TODO: TDI-48980 - Disabling Sonar cache due to "ClassNotFoundException: com.google.common.collect.LinkedListMultimap" issue
```
Could you add the ticket id on same line that the todo for future scan